libr/core/anal_tp.c

```diff
@@ -217,69 +163,146 @@ static int stack_clean (RCore *core, ut64 addr, RAnalFunction *fcn) {
 	free (str);
 	return ret;
 }
+#endif
+
+// Avoid Emulating these instructions
+static inline bool isnonlinear(int optype) {
+	return (optype ==  R_ANAL_OP_TYPE_CALL || optype ==  R_ANAL_OP_TYPE_JMP
```

Comment:
Maybe make a more obvious list of those instead of writing them in one line?

Reply:
As u said in tg :) , it would be a lil slow if we create a list and iterate it to check if it's not-linear indtruction